### ISSUE TITLE
I can't change separators in current file

### ISSUE THREAD

**Issue by zinovevvv** [SeeSharpSoft/intellij-csv-validator#294]
Hi, I try to change separators in current file, but I have no context menu...
<img width="650" alt="image" src="https://user-images.githubusercontent.com/60914683/121726521-552fed00-caf3-11eb-8328-31298b88ef1c.png">
I use PyCharm 2021.1.2 and Plugin 2.16.4

**Comment by SeeSharpSoft** [comment]
As written in the [documentation](https://github.com/SeeSharpSoft/intellij-csv-validator#file-specific-value-separator--escape-character), it's available in the editors context menu (not the project views context menu).

**Comment by gbonline** [comment]
Hi, in Pycharm 2021.2.1, latest update, i don't have CSV entry on the context menu: 
I would like to change the delimiter from , to ; in the file in the background of the image, but no possibility via context menu

![immagine](https://user-images.githubusercontent.com/3273873/133923567-bd3e33b7-d1f0-4609-b517-1b72a53e6930.png)

**Comment by SeeSharpSoft** [comment]
@gbonline It seems the file is (no longer?) associated with the plugin, else the file icon would look different. Please check `File -> Settings -> File Types -> CSV file`, and adjust the file name pattern if needed.

**Comment by gbonline** [comment]
Thanks for answer
I've installed all the plugins suggested from Pycharm CSV, xlsx and Rainbow csv.
Then I've seen that extension did not associated to the CSV plugin but to the other Rainbow CSV plugin. On these configuration I unchecked the checkbox to associate to rainbow csv.
but I forgot to restart pycharm, I tought that it would have been enough.
Today I restarted Pycharm and all goes well: context menu show plugin options and rainbow colors the text.
best regards
